docs: all-flash mode (#129)
iomesh/docs: docs/deploy/prerequisites.md

---

## Installation Requirements

> #### Number of Cluster Nodes

Reviewer: Should be `Kubernetes Cluster Requirement` ? The content contains not only requirement for nodes number

Author: Number of Cluster Nodes  和 Disk Requirements 是平级的标题， Number of Cluster Nodes 内只描述节点数量

Reviewer 2: `Number of Cluster Nodes` 这个作为标题确实比较奇怪，我也倾向于 `Kubernetes Cluster Requirement`

> #### Kubernetes Cluster Requirement

Reviewer 3: Requirement是否需要改为复数Requirements？

Author: done

> A Kubernetes (from v1.17 to v1.21) cluster with at least 3 worker nodes

Reviewer 2: missing `.`

#### 磁盘需求
##### 缓存盘
* 全闪架构: 无需配置

> * 混闪架构: 至少一块空闲的 SSD 磁盘，SSD 磁盘容量大于 60 GB

Reviewer 2: 架构 -> 模式

##### 数据盘
* 全闪架构: 至少一块空闲的 SSD 磁盘，SSD 磁盘容量大于 60 GB

> * 混闪架构: 至少一块空闲的 HDD 磁盘，HDD 磁盘容量大于 60 GB

Reviewer 2: ditto

> * Hybrid-flash mode: there is at least one free SSD on each worker node, and the SSD capacity is larger than 60 GB.

Reviewer 2: ```suggestion
* Hybrid-flash mode: there should be at least one free SSD on each worker node, while the SSD capacity is larger than 60 GB.
```

> * Hybrid-flash mode: there is at least one free HDD on each worker node, and the HDD capacity is larger than 60 GB.

Reviewer 2: `there is` -> `there should be`

`capacity is` -> `capacity should be`

> * All-flash mode: there should be at least one free SSD on each worker node, and the SSD capacity is larger than 60 GB.
> * Hybrid-flash mode: there should be at least one free HDD on each worker node, and the HDD capacity is larger than 60 GB.

Reviewer 2: ```suggestion
* All-flash mode: there should be at least one available SSD on each worker node, and the SSD capacity should be larger than 60 GB.
* Hybrid-flash mode: there should be at least one available HDD on each worker node, and the HDD capacity should be larger than 60 GB.
```

> #### Network Requirement

Reviewer 3: 同上，Requirement是否需要改为复数Requirements?

> A NIC of 10GbE or above is required for the IOMesh storage network.

Reviewer 2: ```suggestion
Network cards of 10GbE or above are required for the IOMesh storage network.
```

#### Reserved System Space

> A space of at least 100 GB is reserved in the /opt directory of each worker node for storing the IOMesh cluster metadata.

Reviewer 2: `A space of at least 100 GB is` -> `100 GB of free disk space is`

Reviewer 2: `of each worker node` -> `on each worker node`

> 100 GB of free disk space is required in the /opt directory on each worker node for storing the IOMesh cluster metadata.

Reviewer 4: 这样呢？
At least 100GB of disk space is required in the /opt directory of each worker node to store IOMesh cluster metadata.

> 100 GB of free disk space is required in the /opt directory on each worker node for storing the IOMesh cluster metadata.

Reviewer 3: ```suggestion
100 GB of free disk space is required in the /opt directory on each worker node for storing the IOMesh cluster metadata. 可以的，看看100 GB是不是要空一格~
```

> ## Setup Worker Node

Reviewer 3: Worker Node Setup
